packages/flagship/src/lib/__tests__/ios.test.ts

```diff
@@ -53,17 +53,17 @@ test(`enable entitlements`, () => {
   });
 
   const entitlementsFileSource = fs
-    .readFileSync(nodePath.join(tempRootDir, `env/${appName}.entitlements`))
+    .readFileSync(nodePath.join(tempRootDir, `env`, `${appName}.entitlements`))
     .toString();
   const entitlementsFile = fs
-    .readFileSync(nodePath.join(tempRootDir, `ios/${appName}/${appName}.entitlements`))
+    .readFileSync(nodePath.join(tempRootDir, `ios`, appName, `${appName}.entitlements`))
     .toString();
   const pbxprojFile = fs
-    .readFileSync(nodePath.join(tempRootDir, `ios/${appName}.xcodeproj/project.pbxproj`))
+    .readFileSync(nodePath.join(tempRootDir, `ios`, `${appName}.xcodeproj`, `project.pbxproj`))
     .toString();
 
   expect(entitlementsFileSource).toEqual(entitlementsFile);
-  expect(pbxprojFile).toMatch(`CODE_SIGN_ENTITLEMENTS = ${appName}/${appName}.entitlements`);
+  expect(pbxprojFile).toMatch(`CODE_SIGN_ENTITLEMENTS = ${appName + nodePath.sep + appName}.entitlements`);
```

Comment:
@Cauldrath The linter is complaining about this line. Can you either split this to two lines or disable the `ter-max-len` rule for this line?